ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -513,6 +513,24 @@ AHU_DFSS_DSP_DX2ZC_ECONZ_EFSS_HTZC:
   - EFSS
   - HTZC
 
+AHU_SFSS_SFVSC_CHWSC_RTM_FDPM2X_ESPC_CO2DFVSC_SSPC2X_CO2M6X_ZTM_VOCM_PMM:
+  description: "Single zone AHU."
+  is_canonical: true
+  implements:
+  - AHU
+  - SFSS
+  - SFVSC
+  - CHWSC
+  - RTM
+  - FDPM2X
+  - ESPC
+  - CO2DFVSC
+  - SSPC2X
```

Comment:
Why would a single-zone unit control to static pressure? This doesn't make sense on this unit. Justify.